shell_integration/dolphin/ownclouddolphinpluginhelper.cpp

```diff
@@ -125,17 +124,19 @@ void OwncloudDolphinPluginHelper::slotReadyRead()
                 return;
             }
         } else if (line.startsWith("V2/GET_CLIENT_ICON_RESULT:")) {
-            line.remove(0, QString("V2/GET_CLIENT_ICON_RESULT:").size());
+            line.remove(0, QStringLiteral("V2/GET_CLIENT_ICON_RESULT:").size());
             QJsonParseError error;
             auto json = QJsonDocument::fromJson(line, &error).object();
-            if (error.error != QJsonParseError::NoError)
+            if (error.error != QJsonParseError::NoError) {
```

Comment:
Please log the error.
.....
Logs like we don't have any logging in here.....

Reply:
Can you add something like 
Q_LOGGING_CATEGORY(lcDolphinExtension, "owncloud.dolphin", QtInfoMsg)

See https://github.com/owncloud/client/blob/master/src/gui/socketapi/socketapi.cpp#L188
and the use qCWarning etc.

Reply:
logging added - but not sure whether something is missing